Data/Streaming/Zlib.hs

```diff
@@ -267,11 +277,16 @@ flushInflate = finishInflate
 --
 -- Since 0.1.11
 getUnusedInflate :: Inflate -> IO S.ByteString
-getUnusedInflate (Inflate (fzstr, _) ref _) = do
+getUnusedInflate (Inflate (fzstr, _) ref _ _) = do
     bs <- readIORef ref
     len <- withForeignPtr fzstr c_get_avail_in
     return $ S.drop (S.length bs - fromIntegral len) bs
 
+-- | Returns True if the inflater has reached end-of-stream, or False if
+-- it is still expecting more data.
```

Comment:
Can you add a `@since`, bump the minor version in the cabal file, and add a note to the ChangeLog? Otherwise looks great, thanks!